Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'plek', '~> 1.11.0'
 gem 'airbrake', '~> 4.3.1'
 
 gem 'gds-sso', '~> 11.0.0'
-gem 'gds-api-adapters', '~> 24.4.0'
+gem 'gds-api-adapters', '~> 25.1'
```

Comment:
We made some [breaking changes and improvements](https://github.com/alphagov/gds-api-adapters/pull/393) to the test helper in 26.0.0, so its probably worth upgrading to that instead.